.evergreen/.evg.yml

```diff
@@ -584,6 +605,13 @@ buildvariants:
   tasks:
      - name: "gssapi-auth-test"
 
+- matrix_name: "perf"
```

Comment:
Currently this will run on every commit but if it's too much we can set batchtime

Reply:
Lets do that

Reply:
Done